zebra-state/src/service/finalized_state.rs

```diff
@@ -53,7 +53,7 @@ pub use disk_db::{DiskWriteBatch, WriteDisk};
 ///
 /// This is different from `NonFinalizedState::clone()`,
 /// which returns an independent copy of the chains.
-#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
It is important here to remove this or we can't add the type `elasticsearch::Elasticsearch` as a field otherwise.

Fortunately, this two derivations were not used at all in current Zebra.